lib/resources/Disputes.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'disputes',
+
+  includeBasic: [
+    'list', 'retrieve', 'update', 'setMetadata', 'getMetadata'
```

Comment:
Yep, that sounds in line with how some of the other libraries do it. In that case, can you remove the metadata methods, too?